source/includes/_01-your-first-backend-in-10-minutes.md

```diff
@@ -0,0 +1,204 @@
+```typescript
+public handle(register: Register): void {
+    register.events(new PostCreated(this.postId, this.title, this.content, this.author))
+  }
+```
+
+## 4. First entity
+We have now created a command and an event, however, we do not have any data representation of a `Post`. As a result, we will create an `entity`.
+
+`boost new:entity Post --fields title:string content:string author:string --reduces PostCreated`
+
+Another file has been created in your project, `<project-root>/src/entities/Post.ts`
+
+The `Post` entity has been created but no projection has been defined, so we will do so:
+
+```typescript
+@Reduces(PostCreated)
+public static projectPostCreated(event: PostCreated, currentPost?: Post): Post {
+return new Post(event.postId, event.title, event.content, event.author)
+}
+```
+
+In the future, we may want to project events for this `Post` entity that require retrieving current `Post` values. In that case we would need to make use of `currentPost`. 
+
+## 5. First read model
+Almost everything is set-up. We just need to provide a way to view the `Posts` of our blog. For that, we will create a `read model`.
+
+`boost new:read-model PostReadModel --fields title:string content:string author:string --projects Post`
+
+Once the read-model code has been generated, it will be placed in `<project-root>/src/read-models/PostReadModel.ts`
+
+Similar to the `CreatePost` command we defined at the beginning, we will also need to define a few things:
+- Who can read the `Posts`
+- How the data is manipulated before returning it
+
+To make it easy, we will allow anyone to read it:
+```typescript
+@ReadModel({
+  authorize: 'all'// Specify authorized roles here. Use 'all' to authorize anyone
+})
+```
+
+and we will project the whole entity
+
+```typescript
+@Projects(Post, "id")
+  public static projectPost(entity: Post, currentPostReadModel?: PostReadModel): PostReadModel {
+    return new PostReadModel(entity.id, entity.title, entity.content, entity.author)
+  }
+```
+
+## 6. Deployment
+Everything we need for a basic project is set. It is time to deploy it:
+
+`boost deploy -e production`
+
+It will take a couple of minutes to deploy all the resources.
+
+When the the serverless backend is successfully deployed you will see information about your stack endpoints. For this basic project we will only need to pick the REST API endpoint, reflected in the output as `backend-application-stack.baseRESTURL`, e.g.:
+
+`https://<api-gateway-id>.execute-api.us-east-1.amazonaws.com/production/`
+
+We will use this endpoint to test our backend.
+
+## 7. Testing
+Let's get started testing the project. We will perform two actions:
+- Add a couple of posts
+- Retrieve all posts
+- Retrieve a specific post
+
+To perform the HTTP requests, you might want to use something like `Postman`, although `curl` would also work.
```

Comment:
I'd go with GraphQL directly, as it will be the main API, unless the other one requires to be prioritized. Also it gives more advantages. I'd add a link to postwoman here, as its a nice tool to test GraphQL.